app/src/main/res/layout/view_search.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
you don't need this xml because of you can customize your edit text
it will take less code and will be more efficient